src/jobs/update_cards.ts

```diff
@@ -313,7 +313,7 @@ function convertColors(card: ScryfallCard, preflipped?: boolean) {
     }
 
     // TODO: handle cards with more than 2 faces
-    return Array.from(card.card_faces[1].colors);
+    return Array.from(card.card_faces[1].colors!);
```

Comment:
TS was being weird. Clearly we have already returned above if card_faces aren't set or if less than 2